### PR TITLE
fix(instance): add support for handling empty security-groups tags

### PR DESCRIPTION
**Pull request by Monitob**

### CODE REVIEW

File: scaleway/resource_instance_security_group_test.go [hunk 1]
```diff
@@ -354,7 +350,6 @@ func TestAccScalewayInstanceSecurityGroup_RemovePort(t *testing.T) {
 			{
 				Config: `
 					resource "scaleway_instance_security_group" "base" {
-						tags = [ "test-terraform" ]
```
Comment (remyleone) [inline scaleway/resource_instance_security_group_test.go:357]: Why are you changing this test? This is precisely what we want to test. Maybe you can add a test with:
- no tags
- adding tags
- modifying tags
- deleting tags

Comment (Monitob) [reply scaleway/resource_instance_security_group_test.go:357]: Yes, I've just added a dedicated test only for tags on `TestAccScalewayInstanceSecurityGroup_Tags` and also for the other resource_instance_XXX with tags

Comment (Monitob) [reply scaleway/resource_instance_security_group_test.go:357]: I've just added new test for this

File: scaleway/resource_instance_security_group_test.go [hunk 2]
```diff
@@ -38,7 +38,6 @@ func TestAccScalewayInstanceSecurityGroup_Basic(t *testing.T) {
 					resource "scaleway_instance_security_group" "base" {
 						name = "sg-name"
 						inbound_default_policy = "drop"
-						tags = [ "test-terraform" ]
```
Comment (Monitob) [inline scaleway/resource_instance_security_group_test.go:41]: @remyleone  here I've removed to test the update of tags.

**Review by remyleone — APPROVED** [review]

**Review by remyleone — CHANGES_REQUESTED** [review]
Fix ci

**Review by remyleone — APPROVED** [review]

**Pull request merged**